src/server/server.cc

```diff
@@ -1091,15 +1090,15 @@ void Server::SetLastRandomKeyCursor(const std::string &cursor) {
 }
 
 int64_t Server::GetCachedUnixTime() {
-  if (unix_time.load() == 0) {
+  if (unix_time_secs.load() == 0) {
     updateCachedTime();
   }
-  return unix_time.load();
+  return unix_time_secs.load();
 }
 
 int64_t Server::GetLastBgsaveTime() {
   std::lock_guard<std::mutex> lg(db_job_mu_);
-  return last_bgsave_time_ == -1 ? start_time_ : last_bgsave_time_;
+  return last_bgsave_timestamp_secs_ == -1 ? start_time_secs_ : last_bgsave_duration_secs_;
```

Comment:
Should `last_bgsave_duration_secs_` be replaced with `last_bgsave_timestamp_secs_` in this line? I mean, we return the timestamp not the duration.